### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sahanihit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+
+Till now the languages used here are:-
+
+  1. **C++**
+  2. **GO**
+  3. **JAVA**
+  4. **PYTHON**
+  
+## C++
+This folder contains 1. [**Modular Exponentiation**](https://github.com/dscjntuc/Data-Structures-and-Algorithms/blob/master/C%2B%2B/Modular%20Exponentiation.cpp) algorithm.
```
Comment (manisaiprasad) [inline README.md:12]: Create a list to show programs in the language folder

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
-# Data-Structures-and-Algorithms-
\ No newline at end of file
+# Data-Structures-and-Algorithms-
```
Comment (manisaiprasad) [inline README.md:1]: Refer to updated Readme

**Review by manisaiprasad — COMMENTED** [review]
Thanks for your interest
Please refer to the updated changes

**Comment by sahanihit** [comment]
Do I have to create a md file in the respective languages folder ?

**Comment by manisaiprasad** [comment]
no need for that 
What I meant is like this

Python algorithms we have 
 * [**Binary Search**](https://github.com/sahanihit/Data-Structures-and-Algorithms/blob/master/Python/Binary%20search.py) algorithm.
* [**Linear Search**](https://github.com/sahanihit/Data-Structures-and-Algorithms/blob/master/Python/LinearSearch.py) algorithm.
* [**Selection Sort**](https://github.com/sahanihit/Data-Structures-and-Algorithms/blob/master/Python/selection%20sort.py) algorithm.

**Comment by sahanihit** [comment]
Ok. Can you please assign it to me? 
I'll do the needful in 1 hour.

**Comment by manisaiprasad** [comment]
Sure @sahanihit

**Pull request closed without merging**